src/plugins/intel_npu/tests/functional/core_config.cpp

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+
+void core_configuration(ov::test::SubgraphBaseTest* test) {
```

Comment:
This file does not seem related to behavior tests.

Reply:
This file seems to be a needed dependency for <code><a href="https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/shared_test_classes/CMakeLists.txt#L5">sharedTestClasses</a></code> lib. <code><a href="https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp#L96">core_configuration</a></code> function is expected to be defined in each plugin I guess, like it is in <code><a href="https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/shared_tests_instances/core_config.cpp#L10">intel_npu</a></code>.

`sharedTestClasses` is in turn a dependency for <code><a href="https://github.com/MirceaDan99/openvino/blob/EISW-120273/copy_npu_func_tests_to_ov/src/plugins/intel_npu/tests/functional/CMakeLists.txt#L29">openvino::funcSharedTests</a></code> and without `openvino::funcSharedTests`, build would fail with errors like:
```
/home/mirceaau/Work/openvino/src/plugins/intel_npu/tests/functional/behavior/elf_config.cpp:5:10: fatal error: 'base/ov_behavior_test_utils.hpp' file not found
#include <base/ov_behavior_test_utils.hpp>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
ninja: build stopped: subcommand failed.
```

Reply:
Understood, thanks!